igvm_defs/src/lib.rs

```diff
@@ -375,6 +375,10 @@ pub enum IgvmPlatformType {
     SEV_SNP = 0x02,
     /// Intel TDX.
     TDX = 0x03,
+    /// AMD SEV.
+    SEV = 0x04,
```

Comment:
hmm is it possible to gate these under the unstable feature since they're part of V2? Otherwise if we can't, it's probably okay.

Reply:
thanks!